pages/docs/reference/java-to-kotlin-interop.md

```diff
@@ -397,53 +397,26 @@ public class BB8 implements Robot {
 ```
 </div>
 
-For the `@JvmDefault` annotation to take effect, the interface must be compiled with an `-Xjvm-default` argument.
-Depending on the case of adding the annotation, specify one of the argument values:
-
-* `-Xjvm-default=enabled` should be used if you add only new methods with the `@JvmDefault` annotation.
-   This includes adding the entire interface for your API.
-* `-Xjvm-default=compatibility` should be used if you are adding a `@JvmDefault` to the methods that were available in the API before.
-   This mode helps avoid compatibility breaks: all the interface implementations written for the previous versions will be fully compatible with the new version.
-   However, the compatibility mode may add some overhead to the resulting bytecode size and affect the performance.
-
-For more details about compatibility issues, see the `@JvmDefault` [reference page](/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default/index.html).
+### Compatibility mode for default methods
 
-### Using in delegates
+If there is client Java code that uses your Kotlin interfaces compiled with Kotlin 1.3 or older, then recompiling it with 
```

Comment:
Main point of compatibility modes is binary compatibility, not source one. (But it also could be used as source one, but it's minor compare to binary one)

Reply:
And it's applies for Kotlin and Java clients